### PR TITLE
cli flag to specify chromedriver port

### PR DESCRIPTION
**Pull request by svkrclg**
https://github.com/sitespeedio/sitespeed.io/issues/3427

### CODE REVIEW

**Comment by svkrclg** [comment]
I am new in this org as a contributor, let me know what else needs to be added.

**Comment by soulgalore** [comment]
Hi @svkrclg looks great, thanks the addition! I'm on vacation, that's why the late merge.

File: lib/chrome/webdriver/builder.js [hunk 1]
```diff
@@ -25,6 +25,9 @@ module.exports.configureBuilder = function(builder, baseDir, options) {
     }
 
     const serviceBuilder = new chrome.ServiceBuilder(chromedriverPath);
+    if (options.chrome.chromedriverPort) {
```
Comment (soulgalore) [inline lib/chrome/webdriver/builder.js:28]: This need either use lodash.get or you can add a check like:
if (options.chrome  && options.chrome.chromedriverPort) ...

The reason is that even when you run Chrome, we aren't 100% you have set any options for Chrome (that's the way it works right now at least).

**Review by soulgalore — CHANGES_REQUESTED** [review]
One minor thing!

File: lib/chrome/webdriver/builder.js (hunk 1 above)
Comment (svkrclg) [reply lib/chrome/webdriver/builder.js:28]: Yes, sure thing.

**Comment by svkrclg** [comment]
Also, do you think it is a good idea to check whether the port is free or not before starting chromedriver?

**Comment by soulgalore** [comment]
@svkrclg no I think that could up to the developer that starts the driver just as long as we push an error to the log if it fails.

**Comment by svkrclg** [comment]
In case of an invalid port, the Chromedriver error is pushed to `chromedriver.log `if verbose >=2

**Comment by soulgalore** [comment]
Thank you @svkrclg !!!

**Pull request merged**